src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -43,8 +43,23 @@
     <ProjectReference Include="..\..\Blazor\DevServer\src\Microsoft.AspNetCore.Blazor.DevServer.csproj" />
     <ProjectReference Include="..\testassets\BasicTestApp\BasicTestApp.csproj" />
     <ProjectReference Include="..\testassets\TestServer\Components.TestServer.csproj" />
+    <ProjectReference Include="..\..\Blazor\testassets\Wasm.Authentication.Server\Wasm.Authentication.Server.csproj" />
   </ItemGroup>
 
+  <Target Name="CopyReferencedProjectStaticWebAssetsManifest" BeforeTargets="Compile">
```

Comment:
Could you clarify why this is needed? I thought that SWA was meant to work equally whether the thing you're referencing is a project or a package, so I'm unsure why we need special handling for our E2E tests.

Reply:
got rid of this